<!DOCTYPE HTML>
<!--
	Story by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Minyue (Camille) Fan</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper" class="divided">

				<!-- Intro -->
					<section class="banner style1 orient-left content-align-left image-position-right fullscreen onload-image-fade-in onload-content-fade-right">
						<div class="content">
							<h1>Minyue (Camille) Fan</h1>
							<p style="font-size:100%;"class="major">I am a recent graduate from Carnegie Mellon University, where I majored in Statistics and Machine Learning. <br> Deciphering data to uncover hidden insights excites me. At CMU, my excitement to learn at a higher level motivated me to take courses at the graduate level while excelling in undergraduate coursework to equip myself with such skills. <br> Outside of coding, I enjoy playing violin, chess and paper-cutting. I am also an orange belt Aikido-<i>ka</i> and a Kendo practitioner. </p>
							<!--<ul class="actions stacked">
								<li><a href="https://drive.google.com/file/d/1xAdOjF04oOk0CSTuqVJ42WWTh_5PeiPR/view?usp=sharing" class="button big wide smooth-scroll-middle">Résumé</a></li>
							</ul>
							<ul class="actions stacked">
								<li><a href="#first" class="button large wide smooth-scroll-middle">Get Started</a></li>

							</ul>-->
							<a href="https://drive.google.com/file/d/1dCh68jzSU5ylxxkX8lOnXEe6WBRlgqnH/view?usp=sharing" 
							class="button big wide smooth-scroll-middle" 
							style="margin:10px">Résumé</a>
							<a href="#first" class="button big wide smooth-scroll-middle" style="margin:10px">Get Started!</a>
							

						</div>


						<div class="image">
							<img src="images/propic.jpeg" alt="" />
						</div>
					</section>

				<!-- Education -->
					<section class="spotlight style1 orient-right content-align-left image-position-center onscroll-image-fade-in" id="first">
						<div class="content">
							<h2>Education</h2>
							<p>
							M.S. in Statistical Practice, Carnegie Mellon University, <i>May 2021 (Expected)</i><br>
							B.S. in Statistics & Machine Learning, Carnegie Mellon University, <i>Dec 2019</i></p>
						</div>
						<div class="image">
							<img src="images/cmu_logo.png" alt=""/>
						</div>
					</section>

				<!-- Programming Languages -->
					<section class="spotlight style1 orient-left content-align-left image-position-center onscroll-image-fade-in">
						<div class="content">
							<h2>Programming Languages</h2>
							<p> Python: <font size="3">TensorFlow, pandas, Pytorch, Keras, NumPy, Jupyter, spaCy, NLTK, scikit-learn, Matplotlib</font> <br> R, C, SQL, Java, Standard Meta Language (SML), Visual Molecular Dynamics (VMD)</p>
						</div>
						<div class="image">
							<img src="images/coding.png" alt="" />
						</div>
					</section>

				<!-- Languages -->
					<section class="spotlight style1 orient-right content-align-left image-position-center onscroll-image-fade-in">
						<div class="content">
							<h2>Languages</h2>
							<p> English, Chinese, Japanese, French, Latin</p>
						</div>
						<div class="image">
							<img src="images/languages.jpeg" alt="" />
						</div>
					</section>

				<!-- Work Experience -->
					<section class="wrapper style1 align-center">
						<div class="inner">
							<h2>Work Experience</h2>
						</div>

						<!-- Gallery -->
							<div class="gallery style2 medium lightbox onscroll-fade-in">
								<article>
									<a href="images/lm.jpeg" class="image">
										<img src="images/lm.jpeg" alt="" />
									</a>
									<div class="caption">
										<h3>Backend Engineer - AI</h3>
										<p> Little Moochi <br> Dec 2019 - Present</p>
									</div>
								</article>
								<article>
									<a href="images/psc.png">
										<img src="images/psc.png" alt="" height = 390/>
									</a>
									<div class="caption">
										<h3>Deep Learning Research Intern</h3>
										<p>Pittsburgh Supercomputing Center <br> May 2019 - May 2020</p>
									</div>
								</article>
								<article>
									<a href="images/kyodai.jpg" class="image">
										<img src="images/kyodai.jpg" alt="" />
									</a>
									<div class="caption">
										<h3>Amgen Scholar</h3>
										<p>Ogata Lab, Kyoto Univeristy <br> May - Aug 2018</p>
									</div>
								</article>
								<article>
									<a href="images/genewiz.png" class="image">
										<img src="images/genewiz.png" alt="" />
									</a>
									<div class="caption">
										<h3>Bioinformatics Intern </h3>
										<p>Genewiz <br> May - Aug 2017</p>
									</div>
								</article>
								
							</div>

					</section>

					<section class="wrapper style1 align-center">
						<div class="inner">
							<div class="index align-left">

								<!-- Text -->
									<section>
										<header>
											<h3>Little Moochi</h3>
											<h4>Backend Engineer - AI</h4>
											<h4><i>Dec 2019 - Present</i></h4>
										</header>
										<div class="content">

											<p>• Lead back-end development at student startup, <a href="https://www.littlemoochi.com/">Little Moochi</a>, to expand product which generated 500+ users since February launch<br>
											• Construct a convolutional neural network with Python that detects ingredients in users’ food images<br>
											• Build an NLP model with skip-thought vectors to match recipes with corresponding images for ingredient analysis<br>
											• Develop nutrition database for food plate classification to provide users with food suggestions based on eating style<br>
											• Collaborate with front-end developers and graphic designers to understand users’ needs and enhance user experience through functional and cohesive codes</p>
										</div>
									</section>

									<section>
										<header>
											<h3>Pittsburgh <br> Supercomputing Center</h3>
											<h4>Deep Learning Research Intern</h4>
											<h4><i>May 2019 - May 2020</i></h4>
										</header>
										<div class="content">

											<p>• Automated the detection of axons in zebrafish microscopy data with convolutional neural networks in TensorFlow <br>
												• Experimented with modifying network structure, rotating cells and parameter testing to improve accuracy to 95% <br>
												• Presented during symposium to 40+ professors and students at Pittsburgh Supercomputing Center<br>
												• Submitted a 8-page research paper to PEARC ’20 and facilitated a virtual presentation<br></p>
										</div>
									</section>

									<section>
										<header>
											<h3>Ogata Lab, <br>Institute for Chemical Research, <br>Kyoto University</h3>
											<h4>Amgen Scholar</h4>
											<h4><i>June - Aug 2018</i></h4>
										</header>
										<div class="content">

											<p>Project: Insight into the role of glycosyltransferases encoded in PkV-RF01 using sequence-based comparison with other viruses<br> 
											• Was selected as one of the 23 Amgen Scholar at Kyoto University among over 400 applicants;<br> 
											• Manually manipulated CAZydb, a database dedicated for Carbohydrate-Active Enzymes annotation from experiments and literature, to compare PkV-RF01 against all known carbohydrate-active enzymes;<br> 
											• Employed HMMER to annotated CAZyme domain boundaries determination according to the dbCAN CAZyme domain HMM database;<br> • Extracted protein sequence of viruses of interests, including PkV-RF01, from Virus-Host Database with Python to perform a homology search<br> 
											•Employed dbCAN to search protein sequences in PkV-RF01 against glycosyltransferases' HMM profile with e-value = 10-5,  coverage = 0.35 to find significant hits;<br> 
											•Mapped protein sequences from previous steps and filtered output to obtain only glycosyltransferases-related genes;<br> 
											• Searched genes against known viruses to infer functions of glycosyltransferases-related genes in PkV-RF01;<br> 
											• Presented poster and oral presentation at symposium to Amgen Scholars and professors from Kyoto University and the University of Tokyo</p>
										</div>
									</section>

									<section>
										<header>
											<h3>Genewiz</h3>
											<h4>Bioinformatics Intern</h4>
											<h4><i>May - Aug 2017</i></h4>
										</header>
										<div class="content">

											<p>• Developed innovative bioinformatics software, algorithms and statistical methods to detect how non-coding RNAs are involved in developing disease <br>
											• Derived insight through statistical analysis including PCA and SVM of multi-level data with R<br>
											• Collaborated with biologists, NGS technologists and software developers across the company</p>
										</div>
									</section>

				

							</div>
						</div>
					</section>

				<!-- Projects -->
					<section class="wrapper style1 align-center">
						<div class="inner">
							<h2>Projects</h2>
							<div class="items style1 medium onscroll-fade-in" align = 'left'>
								<section>
									<span class="icon style2 major fa-gem"></span>
									<h3>Wikipedia Article Question & Answer System</h3>
									<p>• Prepared pipelines using SpaCy for Named Entity Recognition to detect question types <br> • Used PyTorch in building a Recurrent Neural Network for sentiment analysis <br> • Constructed sentence embeddings using GloVe for semantic modeling<br>• Applied tokenization, POS tagging and dependency parsing for text processing and feature extraction <br>• Used NLTK parse tree to compare sentence structures with question and TF-IDF to rank sentence relevance</p>
                                    <ul class="actions stacked"><li><a href="https://www.youtube.com/watch?v=M4sZqee2yX8&t=6s" class="button">Learn More</a></li>
                            </ul>
								</section>
								<section>
									<span class="icon solid style2 major fa-save"></span>
									<h3>Classifying Asians Using Face Recognition by Fine-Grained Deep Learning</h3>
									<p>• Cleaned 54,852 prelabeled profile pictures of Twitter users from China, Japan and Korea and resized images with OpenCV to capture important facial structures and features for an efficient training process <br>• Built a Convolutional Neural Network with Keras containing a pretrained 16-layer VGG with 3 fully connected layers <br>• Achieved 62.3% classification accuracy versus 33.33% by chance and 49% human accuracy</p>
									<ul class="actions stacked"><li><a href="https://drive.google.com/file/d/1HaO_awKMK-xRDmWjypgIv6gQGfIZ7uS_/view?usp=sharing" class="button">See Project</a></li>
								</section>
								<section>
									<span class="icon solid style2 major fa-chart-bar"></span>
									<h3>What Makes Shakespeare Distinctive? A Textual Analysis of Oxford Dictionary of National Biography</h3>
									<p>• Converted words to numerical vectors as word embeddings for quantitative analysis in Python <br>• Compared words and usages in Shakespeare with different time periods using linear transformation of word vectors <br>• Analyzed word context using Principal Component Analysis and K-Means Clustering to determine distinct properties of Shakespeare’s language</p>
									<ul class="actions stacked"><li><a href="https://docs.google.com/presentation/d/1ep7eDmFio1WYsdpuXy7l6yinMxUVZWR57Jb4X8mWhH4/edit?usp=sharing" class="button">See Poster</a></li>
								</section>
								<section>
									<span class="icon solid style2 major fa-wifi"></span>
									<h3>Detecting Alternate Start Codons in RNA</h3>
									<p>• Identified positional information of the main annotated start codons with Python of over 15,000 genes to look for features shared among the start codons <br>• Applied Lasso Regression to select the most important features of upstream start codons <br>• Trained machine learning model in R to predict alternative start codons from large ribosome profiling datasets <br>• Improved the efficiency of previous algorithm by reducing runtime by half and achieved 93% accuracy</p>
								</section>
								<section>
									<span class="icon solid style2 major fa-cog"></span>
									<h3>Predicting AirBnb price and review score</h3>
									<p>• Performed EDA to clean data and remove outliers on 5,201 listings. <br>• Regrouped geographic information (latitude-longitude) to the number of nearby landmarks within a 1-mile radius and quantified amenities per listing for feature engineering<br>• Tuned parameters on Random Forest models with 5-fold cross validation for price prediction.<br>• Decreased training and validation MSE to 1372.49 and 5951.704 from 14288 and 14574 respectively from the baseline linear regression model in price prediction.<br>• Used Xgboost for variable selection and hyperparameter search to predict review score (2 levels, 0,1).<br>• Improved accuracy from 0.862 to 0.9206 from the baseline linear regression model in review prediction. </p>
									<ul class="actions stacked"><li><a href="https://drive.google.com/file/d/1tLMhQW_0ttDhDxDNJY4lpiUM-bWCQBk1/view?usp=sharing" class="button">See Project</a></li>
								</section>
							</div>
						</div>
					</section>


				<!-- Relevant Coursework -->
					<section class="wrapper style1 align-center">
						<div class="inner">
							<h2>Relevant Coursework</h2>
								<p>
								
								<a href="https://www.coursicle.com/cmu/courses/STA/36617/">Applied Linear Models</a><br> 

								<a href="https://www.csd.cs.cmu.edu/course-profiles/15-351-Algorithms-and-Advanced-Data-Structures">Algorithms & Advanced Data Analysis</a><br> 

								<a href="http://www.stat.cmu.edu/~siva/462/main.html">Data Mining</a>  <br> 
								<a href="https://deeplearning-cmu-10707.github.io/">Deep Learning</a>  <br> 
								<a href="https://csd.cs.cmu.edu/course-profiles/15-150-Principles-of-Functional-Programming">Functional Programming</a> <br> 
								<a href="https://www.csd.cs.cmu.edu/course-profiles/15-112-fundamentals-programming-computer-science">Fundamentals of Programming and Computer Science </a><br>  <a href="https://www.csd.cs.cmu.edu/course-profiles/15-251-Great-Theoretical-Ideas-in-Computer-Science">Great Ideas in Theoretical Computer Science</a> <br> 
								Introduction to Statistical Inference <br> 
								<a href="http://www.cs.cmu.edu/~mgormley/courses/10601/">Machine Learning (Graduate Level)</a><br>
								Modern Regression <br> 
								<a href="http://demo.clab.cs.cmu.edu/NLP/">Natural Language Processing</a><br> 
								<a href="https://csd.cs.cmu.edu/course-profiles/15-122-principles-imperative-computation">Principles of Imperative Computation</a><br> 
								Probability Theory and Random Processes <br>
								<a href="https://www.coursicle.com/cmu/courses/STA/36650/">Statistical Computing</a><br> 
								</p>


							<div class="index align-left">
					</section>	

							

				<!-- Footer -->
					<footer class="wrapper style1 align-center">
						<div class="inner">
							<ul class="icons">
								<li><a href="https://github.com/minyuef" class="icon brands style2 fa-github"><span class="label">GitHub</span></a></li>
								<li><a href="https://www.linkedin.com/in/minyf/" class="icon brands style2 fa-linkedin-in"><span class="label">LinkedIn</span></a></li>
								<li><a href="mailto:minyf42@gmail.com" class="icon style2 fa-envelope"><span class="label">Email</span></a></li>
							</ul>
							<p>&copy; Minyue Fan. Last update: Aug 2020 </p>
							Design: <a href="https://html5up.net">HTML5 UP</a>.
						</div>
					</footer>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>